.github/workflows/pull-listener-pkg.yml

```
@@ -15,9 +15,9 @@ jobs:
           go-version: '1.21'
           cache: false
       - name: Lint
-        uses: golangci/golangci-lint-action@v3
+        uses: golangci/golangci-lint-action@v4.0.0
```

```suggestion
        uses: golangci/golangci-lint-action@v4
```

Isn`t it possible to just use v4? Or are expected to pin the version to 4.0.0 and then manually update each minor and patch version?

---

No the whitelisting only works with the full version. Already tried that.

---

Yeah, that is the thing - with the v3 it was kind of "experimental" and minor versions became problematic. Tight now the recommendation is to pin the specific one and upgrade as we go (perhaps we will be able to just do the `v4` at some point).